integreat_cms/cms/fixtures/test_data.json

```diff
@@ -3643,6 +3643,7 @@
       "name": "Test Organization",
       "slug": "test-organization",
       "icon": null,
+      "website": null,
```

Comment:
If we want to enforce icon & website, I would also add one to the test data.

Reply:
I could add an Integreat URL, for example, to `webseite` of the test data, but difficult is the icon. 

We currently don't have any test for the media library. To add an icon to the organization test data, I had to create first an MediaFile object, for which a picture should be stored in the folder `integreat_cms/media/` but this folder will not be stored in the git contro :/ Is there another way?

Another question is, whether it is okay that all the organizations will be deleted after the migration because none of them has right now `website` field. Default values may be cool in this case, tohugh I have no idea what value is reasonable 🤔

Reply:
How about copying one static image into the global media library in the [loadtestdata.sh](https://github.com/digitalfabrik/integreat-cms/blob/develop/tools/loadtestdata.sh) script? And then add a media file object to the test data with this path?

As for the default values: I think it would be fine to set them to an empty string, sure that's not what the apps expect, but maybe less invasive than deleting all invalid organization objects... And then we can tell the service team that they should insert the missing information as soon as possible...

Reply:
Thank you for advice 🙂 The branch has been updated.